version/version.go

```diff
@@ -34,6 +34,10 @@ var (
 	BuildTags = ""
 )
 
+func init() {
+	fmt.Println("Hello!")
+}
```

Comment:
Why do we do this? Is it a forgotten debug code?

Reply:
I've added it on purpose. If there was no code change, the build wouldn't kick off.

Reply:
Maybe you can add other small change? Egg adding a new line without runnning `goftmt`

Reply:
or updating a comment (I'm sure there are many comments which can be improved).

Reply:
This PR is a `Draft`. As such, it is not ready for review. Specifically the commit that introduced this change would be reverted before changing this PR's state to `Ready for review`.